.gitignore

```diff
@@ -3,7 +3,6 @@ masterfiles
 enterprise
 nova
 mission-portal
-design-center
```

Comment:
I would suggest wait with removing this line.
Those who already have it checked out, will suddenly see it in `git gui` and other tools.